### DISCUSSION TITLE
hosting/smtp

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#783] [Announcements]
# hosting/smtp

Usługa SMTP jest przeznaczona wyłącznie dla usług działających na lokalnym klastrze kubernetes. Nie posiada endpointu, który pozwala na używanie go poza nim. Serwer SMTP nie wym...

http://localhost:8080/hosting/smtp.html

**Comment by kkrolikowski** [comment]
cool!
